ui/ducks/app/app.ts

```diff
@@ -114,7 +193,7 @@ export default function reduceApp(state = {}, action) {
 
     // modal methods:
     case actionConstants.MODAL_OPEN: {
-      const { name, ...modalProps } = action.payload;
+      const { name, ...modalProps } = action.value;
```

Comment:
Same here, this was changed to use value when it should be payload. See 'showModal' in actions.js